package.json

```diff
@@ -14,5 +15,6 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
-  }
+  },
+  "proxy": "http://localhost:9000"
```

Comment:
This will tell the App to redirect any unknown requests for non-static content to the backend server.